src/main/java/uk/ac/ebi/ena/webin/cli/manifest/ManifestReader.java

```diff
@@ -357,7 +358,17 @@ private void parseJsonManifest(Path inputDir, List<String> lines ) {
                         }
                     });
                 } else {
-                    handleFieldWithAttributes(inputDir, fieldDefinition, field);
+                    // ValueNode is JsonObject
+                    if(fieldName.equalsIgnoreCase("sample")){
+                        // Check if sample alias is present in sample JSON.
```

Comment:
We are not using the alias here, we are not passing it to anyone else,  so I suggest we should try to get the alias only when we need and report the error then if it is missing.